AllReadyApp/Web-App/AllReady/Views/Campaign/Index.cshtml

```diff
@@ -43,8 +44,14 @@
                         <i class="fa" data-bind="css: { 'fa-square-o': !campaigns.showOld(), 'fa-check-square-o': campaigns.showOld }"></i>
                         Include Completed Campaigns
                     </button>
-                    <span data-bind="text: '(' + total + ')' " ></span>
+                    <span data-bind="text: '(' + total + ')' "></span>
                 </div>
+                @if (await userAuthorizationService.IsCampaignManager())

```

Comment:
Campaign Index has a `List<CampaignViewModel>` as its model, what I can do is to create the property that you've suggested and set it's value for one of the items in the collection and check if any of the items has true for this property, am I correct?